layouts/hub.tsx

```diff
@@ -29,23 +28,6 @@ const HubLayout = ({ children }: { children: React.ReactNode }) => {
     return PATHREGEX.test(path);
   };
 
-  useEffect(() => {
```

Comment:
This whole `useEffect` is no longer required as the middleware handles whether or not someone is logged in as well as if they're onboarded or not.